html/html.go

```diff
@@ -137,10 +134,17 @@ func (e *Engine) Load() error {
 		if err != nil {
 			return err
 		}
+		// Allocate template and set template settings
+		if e.Templates == nil {
```

Comment:
Only allocate the template when needed instead of creating an empty template with the name of `e.directory`